docs/book/en/src/contribute/installer.md

```diff
@@ -0,0 +1,151 @@
+* Description: Show this help message and exit.
+
+### Verbose
+* Short Option: `-D`
+* Full Option: `--debug`
+* Description: Enable verbosity debug
+
+### Specify Installed Version
+* Short Option: `-v VERSION`
+* Full Option: `--version VERSION`
+* Description: Install the given VERSION of WasmEdge
+* Available Value: VERSION `0.11.2` or other valid release versions.
+* Note - In case if an invalid or non existing version is suppliead the installer exits with an error.
+
+### Install Path
+* Short Option: `-p PATH`
+* Full Option: `--path PATH`
+* Description: Install WasmEdge into the given PATH. The default Path is `$HOME/.wasmedge`.
+* Note - In any path other than the ones starting with `/usr` are treated as non system paths in the internals of the installer. The consequences are different directory structures for both.
+* Note - If the path does not exist it is created.
+
+### Uninstall existed version
+
+#### Run uninstaller before installing
+* Short Option: `-r {yes,no}`
+* Full Option: `--remove-old {yes, no}`
+* Description: Run the uninstaller script before installing. Default `yes`.
+
+#### Use a specific version of the uninstaller
+* Short Option: `-u UNINSTALL_SCRIPT_TAG`
+* Full Option: `--uninstall-script-tag UNINSTALL_SCRIPT_TAG`
+* Description: Use the given GitHub tag to uninstall the script
+
+### Install Extensions
+* Short Option: `-e [EXTENSIONS [EXTENSIONS ...]]`
+* Full Option: `--extension [EXTENSIONS [EXTENSIONS ...]]`
+* Description: Install wasmedge-extension tools.
+* Available Value (case sensitive): Supported Extensions `'tensorflow', 'image', 'all'`.
+
+#### Tensorflow Extensions Library Version
+* Full Option: `--tf-version TF_VERSION`
+* Description: Install the given VERSION of the library of the Tensorflow and Tensorflow lite extension. Only available when the `Extensions` is set to `all` or `tensorflow`.
+* Note - It's value is same as that of wasmedge version if not specified.
+
+#### Tensorflow Extensions Dependencies Version
+* Full Option: `--tf-deps-version TF_DEPS_VERSION`
+* Description: Install the given VERSION of the dependencies of the Tensorflow and Tensorflow lite extension. Only available when the `Extensions` is set to `all` or `tensorflow`.
+* Note - It's value is same as that of wasmedge version if not specified.
+
+#### Tensorflow Extensions Tools Version
+* Full Option: `--tf-tools-version TF_TOOLS_VERSION`
+* Description: Install the given VERSION of the tools of the Tensorflow and Tensorflow lite extension. Only available when the `Extensions` is set to `all` or `tensorflow`.
+* Note - It's value is same as that of wasmedge version if not specified.
+
+#### Image Extensions Version
+* Full Option: `--image-version IMAGE_VERSION`
+* Description: Install the given VERSION of the Image extension. Only available when the `Extensions` is set to `all` or `image`.
+* Note - It's value is same as that of wasmedge version if not specified.
+
+#### Image Extensions Dependencies Version
```

Comment:
Remove this column.
image-deps has been removed from installer due to deprecating 0.8.2.